runtime/doc/nvim-extra-keymaps.txt

```diff
@@ -0,0 +1,36 @@
+*nvim-extra-keymaps.txt*    For nvim.                         {Nvim}
+						    Last change:      2015 May
+
+EXTRA KEYMAPS FOR NVIM                                      *nvim-extra-keymaps*
+
+The `plugin/extra-keymaps.vim` script provides a few extra keymaps that might
+be useful and tries to fix some issues the community has identified with vim's
+default keymaps.
+
+The user can disable the plugin by having >
```

Comment:
I might have mentioned this before, but maybe just `This plugin can be disabled by adding the following to your |vimrc|:`.